From 048203c97009c907ff3891f6ffa8f375fcf1045c Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 26 Oct 2021 08:34:39 -0700
Subject: [PATCH] crypto: fix build without scrypt

* add missing semicolon to fix:
  In file included from ../src/node_crypto.h:47,
                   from ../src/node.cc:46:
  ../src/crypto/crypto_scrypt.h:80:2:
  error: expected ';' after struct definition
     80 | }
        |  ^
        |  ;

  and fix typo in the comment

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Submitted [https://github.com/nodejs/node/pull/40613]

---
 src/crypto/crypto_scrypt.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/crypto/crypto_scrypt.h b/src/crypto/crypto_scrypt.h
index 4ca888e31d..3d185637f4 100644
--- a/src/crypto/crypto_scrypt.h
+++ b/src/crypto/crypto_scrypt.h
@@ -77,8 +77,8 @@ struct ScryptJob {
   static void Initialize(
       Environment* env,
       v8::Local<v8::Object> target) {}
-}
-#endif  // !OPENSSL_NO_SCRIPT
+};
+#endif  // !OPENSSL_NO_SCRYPT
 
 }  // namespace crypto
 }  // namespace node
